relayer-cli/src/config.rs

```diff
@@ -68,6 +75,19 @@ pub fn validate_config(config: &Config) -> Result<(), Error> {
         validate_trust_threshold(&c.id, c.trust_threshold)?;
     }
 
+    // Check for invalid mode config
+    validate_mode(&config.mode)?;
+
+    Ok(())
+}
+
+fn validate_mode(mode: &ModeConfig) -> Result<(), Error> {
+    if mode.clients.enabled && !mode.clients.refresh && !mode.clients.misbehaviour {
+        return Err(Error::invalid_mode(
+            "either `refresh` or `misbehaviour` must be set to true if `clients.enabled` is set to true".to_string(),
+        ));
+    }
+
```

Comment:
Would the following be considered valid config? 
```toml
[mode]

[mode.clients]
enabled = false

[mode.connections]
enabled = false

[mode.channels]
enabled = false

[mode.packets]
enabled = false
```

Reply:
> @ancazamfir: it’s valid and actually interesting to just watch what’s going on. I would use this :slightly_smiling_face: But a warning is definitely required.

Reply:
https://github.com/informalsystems/ibc-rs/pull/1539/commits/2e3bca3cc9617f102cfff02a56b1aa4bc0264f19